articles/connections/database/password-change.md

```diff
@@ -0,0 +1,68 @@
+---
+url: /password_change
+---
+
+# Changing a User's Password
+
```

Comment:
Myabe add a sentence like "this document explains the various ways you can reset passwords..."